src/interceptors/multipart_request.js

```diff
@@ -1,33 +1,64 @@
-import _ from 'lodash';
+const _ = require('lodash');
+const FormData = require('form-data');
+const fs = require('fs');
 
 /**
-   Takes `params` from `ctx` and converts to `FormData`
+   Wraps FormData's async getLength fn in a Promise
+*/
+const getLengthPromise = formData =>  new Promise((resolve, reject) => {
+  formData.getLength((err, length) => {
+    if (err) {
+      reject(err);
+    }
+    resolve(length);
```

Comment:
This `resolve` is now run also after `reject(err)` is called since there is no `return` in between. It might work fine since I think promises can only be settled (resolved or rejected) once, and further settling is ignored. But maybe add an `else` around the `resolve` call?

Reply:
Good point, I'll add an `else` there.